### DISCUSSION TITLE
nwg-look 0.2.0 (2023-03-07)

### DISCUSSION THREAD

**Discussion by nwg-piotr** [nwg-piotr/nwg-shell#80] [Changelog]
- added fix for `nwg-look -a` to respect color-scheme settings by @MarkusVolk;
- added translation engine;
- added pl_PL lang.

[edit] I had to push a quick update to 0.2.1, due to the `toolbar-icon-size` label missing from pl_PL and written in Polish in en_US. :)

## Packaging

Please note that from now on you need to copy `langs` to `/usr/share/nwg-look/`.

**Comment by mac-a-r0ni** [comment]
Just leaving comment here since it's not really a bug per-se, but on Slackware stable 15.0, this will not build. Golang can't move past 1.19 because of the gcc slackware shipped and the vendor mods call for 1.20 minimum (which I'm seeing Arch is shipping now). This is likely not an issue on the -current slackware dev systems, but I haven't had the time to test yet. We're about to enter a release beta soon enough and very likely in the next 3~ months there will be a new stable release which WILL support Go 1.20+.

This hasn't affected any other Go packages yet, but when it does, they will essentially be frozen at those versions on 15.0. I'll be returning my focus to our "rolling" dev release very soon.

**Reply by nwg-piotr** [reply]
If you can modify go.mod on your side, and **if** I remember well, go>1.16 should be enough.

**Reply by mac-a-r0ni** [reply]
> go>1.16 should be enough.

I'll do some testing and see if I can't get a package out of it after-all!

**Reply by nwg-piotr** [reply]
Thank you. At the moment I'm a little confused by the recent contributions to nwg-panel, that introduced more bugs than features. Need to try and wrap my head around it.

**Reply by mac-a-r0ni** [reply]
I'm sure you'll get to the bottom of it! btw I fixed it, nwg-look now builds again!

**Reply by nwg-piotr** [reply]
Great! I keep on trying to keep up with golang versions, but this language evolves almost as fast as damned Android SDK. 🤣

**Comment by nwg-piotr** [comment]
SlackBuilds packages are now [visible](https://github.com/nwg-piotr/nwg-shell) in repology badges, yay!

**Reply by mac-a-r0ni** [reply]
Nice to see them there!

Side note: I had submitted updates for some packages and they stay pending until approval. But once approved, I can't modify them again for the week. As a result, I can't submit updates for a few packages until next week, since there were more releases after I submitted them. Going forward, I'll likely wait to submit updates until it nears the deadline in case any other releases get published during the week. There's not usually this much activity so I assume it won't be a major issue going forward but things will fluctuate during any active development times.

**Reply by nwg-piotr** [reply]
Haha, we live on the bleeding edge here on Arch. ;) Seriously: the size of the project is starting to overwhelm me. If I don't release a new version right after making changes,  I may forget about them for the next six months.

**Reply by mac-a-r0ni** [reply]
Haha oh I get it. Managing projects with a considerable size things tend to get lost in the shuffle. I usually live in the unstable land of Slackware development versions so I get things immediately as well just like Arch but for those who stay on stable there is a wait time involved in their "stability". I'm generally better at keeping my own repos up to date but I've not been doing so focusing on the SlackBuilds versions, but I'm working on fixing that right now ;) 

I'm working out a script to take me from stable to devel distros without any formatting etc, the biggest hurdle has become e2fsprogs which has an incompatibility with grub since an update recently. Grub needs patching on both ends before things work normally. That's all done now, so now I need to test hotswapping my OS/chroots to see if it all worked out!

**Reply by nwg-piotr** [reply]
Honestly, Slackware is a bit too geeky to my taste. In my 50s', I prefer to focus on my own ideas, instead of maintaining the system itself.  Arch is rock stable, by the way.

**Reply by mac-a-r0ni** [reply]
Well any distro can be stable, if the user knows how to manage it ;)
That said, I don't have any problems overall with it besides not needing or having a use for systemd, and I really don't care for the package manager. The distro itself is fine, but if I had to use one with a overbearing package manager, I'd use Debian. I rather enjoy the freedom I have in Slackware not forcing any dependencies on me, and can make things work for me instead of conforming to someone else's standard on how something should ship. If it weren't for the abysmal hardware support, I'd be on BSD by now. But Slackware does just fine for my old tastes.

**Reply by nwg-piotr** [reply]
I must give BSD a try one day.

> overbearing package manager

I'm quite happy with pacman, except for my own stuff. I use packaged versions on the testing lappy only.
